modules/dr-restoring-cluster-state.adoc

```diff
@@ -281,75 +281,61 @@ etcd-ip-10-0-143-125.ec2.internal                1/1     Running     1
 +
 If the status is `Pending`, or the output lists more than one running etcd pod, wait a few minutes and check again.
 
-. If you are using the `OVNKubernetes` network plugin, delete the node objects that are associated with control plane hosts that are not the recovery control plane host.
+. If you are using the `OVNKubernetes` network plugin, you must restart `ovnkube-controlplane` pods.
+.. Delete all the ovnkube-controlplane pods by running the following command:
 +
 [source,terminal]
 ----
-$ oc delete node <non-recovery-controlplane-host-1> <non-recovery-controlplane-host-2>
+$ oc -n openshift-ovn-kubernetes delete pod -l app=ovnkube-control-plane
 ----
-
-. Verify that the Cluster Network Operator (CNO) redeploys the OVN-Kubernetes control plane and that it no longer references the non-recovery controller IP addresses. To verify this result, regularly check the output of the following command. Wait until it returns an empty result before you proceed to restart the Open Virtual Network (OVN) Kubernetes pods on all of the hosts in the next step.
+.. Deploy all of the ovnkube-controlplane pods again by running the following command:
 +
 [source,terminal]
 ----
-$ oc -n openshift-ovn-kubernetes get ds/ovnkube-master -o yaml | grep -E '<non-recovery_controller_ip_1>|<non-recovery_controller_ip_2>'
+$ oc -n openshift-ovn-kubernetes get pod -l app=ovnkube-control-plane
```

Comment:
Definitely confirm with an SME, but I don't think this command will deploy the pods again. The `get` command just retrieves information.

Though actually now that I think about it, they are probably redeployed automatically because you deleted the pods in the previous command. So you might just need to reword the above step to not say "Deploy", but instead say something like verify that all of the pods are deployed again...

Reply:
It should be `delete`, not `get`. Good catch.

Reply:
@palonsoro The previous command (line 289) does already delete the pods.

This command looks like it might just be trying to verify that the pods were redeployed. So it might just be the wording above the command that needs to be changed (instead of saying "Deploy... " maybe it should say something about verifying that the pods were redeployed?)

Reply:
Ok. I missed the context around. Then yes, you don't deploy but you wait for the pods to be deployed (by OpenShift)